variables.tf

```diff
@@ -333,6 +333,12 @@ variable "cluster_config_endpoint_type" {
   }
 }
 
+variable "enable_ocp_console" {
```

Comment:
needs to be exposed in the fscloud submodule too

Reply:
Done